tracer/src/Datadog.Trace.Tools.dd_dotnet/ExporterSettings.cs

```diff
@@ -0,0 +1,92 @@
+// <copyright file="ExporterSettings.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Configuration.Telemetry;
+using Datadog.Trace.Tools.dd_dotnet;
+
+// ReSharper disable once CheckNamespace - Needed for compatiblity with linked files
+namespace Datadog.Trace.Configuration;
+
+/// <summary>
+/// Lightweight version of ExporterSettings for the tool
+/// </summary>
+public partial class ExporterSettings
+{
+    private readonly Func<string, bool> _fileExists = File.Exists;
+    private readonly DummyTelemetry _telemetry = new();
+
+    private Uri _agentUri;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ExporterSettings"/> class.
+    /// </summary>
+    /// <param name="configuration">Configuration source</param>
+    public ExporterSettings(IConfigurationSource? configuration)
+    {
+        var agentUri = GetValue(configuration, ConfigurationKeys.AgentUri);
+        var tracePipeName = GetValue(configuration, ConfigurationKeys.TracesPipeName);
+        var agentHost = GetValue(configuration, ConfigurationKeys.AgentHost, "DD_TRACE_AGENT_HOSTNAME", "DATADOG_TRACE_AGENT_HOSTNAME");
```

Comment:
I know that you copied this from `dd-trace` tool but, I cannot find those environment variables anywhere else ... I may not have the full context...
shouldn't it be `DD_AGENT_HOST` instead ? in any case, shouldn't `DD_AGENT_HOST` be handled too ?
are the `DATADOG_XXX` env vars some kind of legacy ?